metecho/api/jobs.py

```diff
@@ -291,10 +290,11 @@ def create_repository(
                 raise Exception("Failed to push files to GitHub repository")
 
         # Copy branch protection rules from the template repo
-        if tpl_repo:
-            copy_branch_protection(
-                source=tpl_repo.branch(branch_name), target=repo.branch(branch_name)
-            )
+        # See copy_branch_protection() for why we don't use this currently
```

Comment:
Don't love this having to be commented out, but I understand a 🐛 with GitHub requires it...